x/virtualgroup/keeper/msg_server.go

```diff
@@ -459,40 +480,152 @@ func (k msgServer) StorageProviderExit(goCtx context.Context, msg *types.MsgStor
 func (k msgServer) CompleteStorageProviderExit(goCtx context.Context, msg *types.MsgCompleteStorageProviderExit) (*types.MsgCompleteStorageProviderExitResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	operatorAddr := sdk.MustAccAddressFromHex(msg.StorageProvider)
+	operatorAddr := sdk.MustAccAddressFromHex(msg.Operator)
```

Comment:
Do we have such txs on testnet? If there is, this could be a hardfork.

Reply:
yes, there is one..

Reply:
It should be possible to adjust the order of the messages in `msg`.